index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+  <title>My_Portfolio</title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <header>
+    <div class="top-bar">
+      <img src="/images/Time Style.png">
+      <img src="/images/Group 4.png">
+    </div>
+    <div class="navigation">
+      <p class="logo">Enoch</p>
+      <a href="#"><img src="/images/Enabled.png"></a>
+    </div>
+    <section class="main">
+      <div class="intro">
+        <p class="para1">Hey There.<br> I'm Enoch</p>
+        <p class="para2">I'm a Software Developer</p>
```

Comment:
While you made great effort curating this project, it is a worthwhile effort to use HTML semantic tags. These are mostly regarded as headers. Try and use h1-h6 for these parts of your code.